rust-toolchain

```diff
@@ -1 +1 @@
-1.64.0
+stable
```

Comment:
i think we probably want to update this to a recent static version -- this appears to break fetching rustfmt, at least.